.babelrc

```diff
@@ -0,0 +1 @@
+{ "stage": 1 }
```

Comment:
Umm, Babel 6 is not backwards compatible, you must have Babel 5 running somehow (?). Decorators have been removed from stage 1 and are no in the transform-decorators-legacy plugin. So I guess this could be the config:

```json
{
  "presets": [
    "es2015",
    "react"
  ],
  "plugins": [
    "transform-decorators-legacy"
  ]
}
```

```json
    "babel-core": "^6.9.0",
    "babel-plugin-transform-decorators-legacy": "^1.3.4",
    "babel-preset-es2015": "^6.9.0",
    "babel-preset-react": "^6.5.0",
```